dogu.json

```diff
@@ -32,6 +32,18 @@
     "Name": "forgot_password_text",
     "Description": "Text which shall be shown if a user clicks 'Forgot Password'",
     "Optional": true
+  },{
+    "Name": "legal_urls/terms_of_service",
+    "Description": "This URL links to the Terms of Service and will be be the first link in bottom of login or logout pages. If empty the whole link will be omitted. If changed the CAS dogu must be restarted to take effect.",
+    "Optional": true
+  },{
+    "Name": "legal_urls/imprint",
+    "Description": "This URL links the Imprint and will be be the second link in bottom of login or logout pages. If empty the whole link will be omitted. If changed the CAS dogu must be restarted to take effect.",
+    "Optional": true
+  },{
+    "Name": "legal_urls/privacy_policy",
+    "Description": "This URL links the Privacy Policy and will be be the last link in bottom of login or logout pages. If empty the whole link will be omitted. If changed the CAS dogu must be restarted to take effect.",
```

Comment:
Maybe "This URL links to the Privacy..." and "... last link at the bottom of login..."